scripts/check_frontend_test_coverage.py

```diff
@@ -297,6 +297,7 @@
     'state-interaction-editor.directive.ts',
     'state-property.service.ts',
     'state-responses.directive.ts',
+    'state-skill-editor.directive.ts',
```

Comment:
Please **do not** add files here. See the comment at the top of this list. New files need unit tests.

Reply:
Done.